server/main.js

```diff
@@ -1,41 +1,140 @@
-import { Meteor } from 'meteor/meteor'
-import { LinksCollection } from '/imports/api/links'
+import { Meteor } from 'meteor/meteor';
+import { Random } from "meteor/random";
+import { PlantsCollection } from '/imports/api/plants';
+import { UsersCollection } from '/imports/api/users';
+import { ClimatesCollection } from '/imports/api/climates';
+import { SensorsCollection } from '/imports/api/sensors';
 
-async function insertLink({ title, url }) {
-  await LinksCollection.insertAsync({ title, url, createdAt: new Date() })
-}
+//Reset dbs: db.getCollectionNames().forEach(function(collectionName){if (!collectionName.startsWith("system.")) {db[collectionName].drop();}});
 
 Meteor.startup(async () => {
-  // If the Links collection is empty, add some data.
-  if ((await LinksCollection.find().countAsync()) === 0) {
-    await insertLink({
-      title: 'Do the Tutorial',
-      url: 'https://vuejs.org/guide/quick-start.html',
-    })
-
-    await insertLink({
-      title: 'Follow the Guide',
-      url: 'https://guide.meteor.com',
-    })
-
-    await insertLink({
-      title: 'Read the Docs',
-      url: 'https://docs.meteor.com',
-    })
+  if ((await ClimatesCollection.find().countAsync()) === 0) {
+    await Promise.all([
+      ClimatesCollection.insertAsync({ 
+        name: 'Tempéré', 
+        description: 'Climat modéré avec quatre saisons distinctes',
+        temperatureRange: { min: 5, max: 25 }
+      }),
+      ClimatesCollection.insertAsync({ 
+        name: 'Méditerranéen', 
+        description: 'Climat chaud et sec en été, doux en hiver',
+        temperatureRange: { min: 10, max: 35 }
+      })
+    ]);
+  }
 
-    await insertLink({
-      title: 'Discussions',
-      url: 'https://forums.meteor.com',
-    })
+  if ((await PlantsCollection.find().countAsync()) === 0) {
+    await Promise.all([
+      PlantsCollection.insertAsync({ 
+        name: 'Tomate',
+        maintenances: [
+          {
+            _id: Random.id(),
+            name: "Arrosage",
+            frequency: 5,
+          },
+          {
+            _id: Random.id(),
+            name: "Sulfatage",
+            frequency: 150,
+          }
+        ]
```

Comment:
Maintenances should have their own collection, therefore here, plants should only have a maintenance_id